compiler/noirc_frontend/src/elaborator/expressions.rs

```diff
@@ -437,11 +437,17 @@ impl<'context> Elaborator<'context> {
     ) -> Vec<(Ident, ExprId)> {
         let mut ret = Vec::with_capacity(fields.len());
         let mut seen_fields = HashSet::default();
+        let struct_id = struct_type.borrow().id;
```

Comment:
nit: could you not fetch where it was used for creating the `StructMember` reference?

Reply:
Done! I can't remember why I put it there... 😅